MBBSEmu.Tests/ExportedModules/Majorbbs/isuidc_Tests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class isuidc_Tests : ExportedModuleTestBase
+    {
+        private const int ISUIDC_ORDINAL = 364;
+
+        [Theory]
+        [InlineData(31, 0)]
+        [InlineData('!', 1)]
+        [InlineData('~', 1)]
```

Comment:
You have some weird cases which is nice, but no normal cases like a simple letter :)

Reply:
Since it is a range function I added tests for the range. I went ahead and did some for ASCII values within the range as well.